.circleci/config.yml

```
@@ -334,6 +334,10 @@ commands:
             sudo apt-get update && sudo apt-get --yes install --no-install-recommends yarn=<< parameters.yarn_version >>
             yarn --version
       - *restore_frontend_deps
+      - run:
+          name: Install Playwright dependencies
+          command: |
+            sudo apt-get update && sudo apt-get --yes install --no-install-recommends libgbm
```

Actually, this should be:
```suggestion
            sudo apt-get update && sudo apt-get --yes install --no-install-recommends libgbm1
```